app/views/users/index.html.erb

```diff
@@ -0,0 +1,12 @@
+<ul>
+  <% @users.each do |user| %>
+    <li>
```

Comment:
- [ ] You have done a great job on the view files however please make sure that you also render data from the models in your controllers, otherwise there will be nothing to display when you write @users in the view files.
@users refers to the instance variable declared in the controllers and can be used in the view files to access the data from the models.
Here is an example snippet of how to define and use @users in the controller and the view files:

```ruby
 def index
      @users = User.all
    end

```

Reply:
This change has been done well